### PR TITLE
Sign the CLA (brentburg)

### PR DESCRIPTION
**Pull request by brentburg**

### CODE REVIEW

File: contributors/brentburg.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+2015-05-05
```
Comment (xiao) [inline contributors/brentburg.md]: 2015-05-01?

**Comment by brentburg** [comment]
Fixed

**Comment by xiao** [comment]
Thanks!

**Comment by xiao** [comment]
Could you also squash the commits in this PR?

**Comment by brentburg** [comment]
Done

**Comment by xiao** [comment]
Thanks!

**Pull request merged**